tables/de-g1-bd-core.cti

```diff
@@ -0,0 +1,44 @@
+# liblouis: German grade 1 braille (bidirectional)
```

Comment:
This table should be merged with de-g1-core.cti

Reply:
> I agree, but the lines that are different between the two tables would then have to be moved to the main tables. This is a change to the structure of the original tables, so I think I must have the approval of the owner before doing this refactoring.

You don't need approval of the owner to do structural changes. Especially not in the PR phase.

But anyway, the only lines that are different are the `nofor` lines that you added. So at the very least you can include de-g1-core.cti in de-g1-bd-core.cti. But you should also just be able to add the `nofor` lines to de-g1-core.cti because they won't affect the forward translation.

> Actually, I originally also made the Danish tables that way, with common core files, but I was then asked to split the tables and keep them separate, even though they share most lines, like the German tables.

Who asked you that?

With the Danish tables it's slightly less of an issue I think because all the files are maintained by you anyway, and we know it is done well. But in this case, you are either asking another person to suddenly maintain a lot more files, or you maintain the new table yourself but that means they can easily diverge if no precautions are made. 

> I suggest we keep them separate for now and then make it a milestone for the next release, perhaps both for the German and the Danish Tables.

OK, I can live with that.

Reply:
See https://github.com/liblouis/liblouis/issues/911